### DISCUSSION TITLE
Github actions problem

### DISCUSSION THREAD

**Discussion by burninghering** [MrGreensWorkshop/MrGreen-JekyllTheme#169] [Problems] [answered]
![image](https://user-images.githubusercontent.com/37091602/216990302-5ee479ee-732e-499c-9b8f-c0871ff8dbcd.png)
The blog runs fine, but Github Actions gives me this problem. Is it not a problem because the blog is running properly?

**Comment by MrGreensWorkshop** [comment, marked as answer]
Every time you change something, the workflow runs and compiles your website. If you don't wait until it finishes, what you will see on the Actions page doesn't make any sense. When you change something, check to see if it's a success; if it fails, check the log and correct your mistake, then continue building your website. For long run I recommend you to build locally check if it builds. Then push to your reprosoritory.
